ts/features/idpay/unsubscription/xstate/selectors.ts

```diff
@@ -16,3 +17,6 @@ export const isLoadingSelector = createSelector(selectTags, tags =>
 
 export const selectIsFailure = (state: StateWithContext) =>
   state.matches("UNSUBSCRIPTION_FAILURE");
+
+export const selectInitiativeType = (state: StateWithContext) =>
+  state.context.initiativeType ?? InitiativeRewardTypeEnum.REFUND;
```

Comment:
```suggestion
export const selectInitiativeType = (state: StateWithContext) =>
  pipe(
    state.context.initiativeType,
    O.fromNullable,
    O.getOrElse(() => InitiativeRewardTypeEnum.REFUND)
  );
```
Sorry, I'm a `fp-fetish` 😂